therapi-runtime-javadoc/src/main/java/com/github/therapi/runtimejavadoc/Link.java

```diff
@@ -1,16 +1,19 @@
 package com.github.therapi.runtimejavadoc;
 
+import java.util.Arrays;
 import java.util.Objects;
 
 public class Link {
     private final String label;
     private final String referencedClassName;
     private final String referencedMemberName;
-
-    public Link(String label, String referencedClassName, String referencedMemberName) {
+    private final String[] params;
```

Comment:
To maintain immutability (and consistency with other classes) can you please change this to a `List<String>` and initialize it with `unmodifiableDefensiveCopy`?

Reply:
Done